Mathlib/LinearAlgebra/Basis/Defs.lean

```diff
@@ -364,8 +361,6 @@ def mapCoeffs (h : ∀ (c) (x : M), f c • x = c • x) : Basis ι R' M := by
   letI : Module R' R := Module.compHom R (↑f.symm : R' →+* R)
   haveI : IsScalarTower R' R M :=
     { smul_assoc := fun x y z => by
-        -- Porting note: `dsimp [(· • ·)]` is unavailable because
-        --               `HSMul.hsmul` becomes `SMul.smul`.
         change (f.symm x * y) • z = x • (y • z)
```

Comment:
So, this porting note still "applies" but you declare it wontfix, right? Same a few lines below.

Reply:
Yup, indeed.